synapse/models/orgs.py

```diff
@@ -87,56 +143,27 @@ def getBaseModels():
                 ]),
 
                 ('ou:owns', {'ptype': 'sepr', 'sep': '/', 'fields': 'owner,ou:org|owned,ou:org'}, [
-                ]),
+                ]),  # FIXME does this become an ou:org:has?
```

Comment:
Meant to bring this up for discussion (separately). ou:owns seems duplicative of ou:suborg, with the only real distinction being that ou:suborg can represent "org1 owns org2" (or "org1 bought and is in the process of completely absorbing org2") as well as "org2 is a department or subdivision within org1". I don't see a significant need to make this distinction. Thoughts?

Reply:
...but if we keep it then yes, I think it becomes an ou:org:has and it definitely means "owns as a separate entity" and not department / subdivision / previously acquired and now fully absorbed.

Reply:
Tracking this separately